README.md

```diff
@@ -14,7 +14,7 @@ We plan to land this upstream in ros2_controllers: https://github.com/ros-contro
 ### Usage
 Example of instance within `ros2_control` can be found in the [configuration file](https://github.com/PickNikRobotics/ros2_kortex/blob/main/kortex_description/arms/gen3/7dof/config/ros2_controllers.yaml).
 Defined interfaces have to exist within the `hardware_interface::SystemInterface` implemented for the robot (e.g. driver).
-Stop any commanding controller and then start `TwistController`to use kinova's builtin cartesian twist controller.
+Stop any commanding controller and then start `PicknikTwistController`to forward twist msgs to the hardwares builtin cartesian twist controller.
```

Comment:
No need to block on this but I have 1 small nitpick. Everything else looks good to me.
```suggestion
Stop any commanding controller and then start `PicknikTwistController`to forward twist msgs to the hardware's built-in cartesian twist controller.
```